public/v4/apps/searx.yml

```diff
@@ -0,0 +1,96 @@
+            BIND_ADDRESS: 0.0.0.0:8080
+            BASE_URL: $$cap_SEARX_BASEURL
+            MORTY_URL: $$cap_MORTY_URL
+            MORTY_KEY: $$cap_MORTY_KEY
+        caproverExtra:
+            containerHttpPort: '8080'
+    # Filtron
+    $$cap_appname-filtron:
+        volumes:
+            - $$cap_FILTRON_RULES:/etc/filtron/rules.json
+        ports:
+            - 4004:4004
+            - 4005:4005
+        restart: always
+        caproverExtra:
+            dockerfileLines:
+                - FROM dalf/filtron@$$cap_filtron_version
+                - ENTRYPOINT ["/usr/local/filtron/filtron", "--rules", "/etc/filtron/rules.json", "--listen", "0.0.0.0:4004", "--api", "0.0.0.0:4005", "--target", "srv-captain--searx:8080"]
+            containerHttpPort: '3000'
+    # Morty
+    $$cap_appname-morty:
+        restart: always
+        environment:
+            MORTY_KEY: $$cap_MORTY_KEY
+            MORTY_ADDRESS: 0.0.0.0:3000
+        caproverExtra:
+            containerHttpPort: '3000'
+            dockerfileLines:
+                - FROM dalf/morty@$$cap_morty_version
+                - CMD -listen 0.0.0.0:3000 -timeout 6 -ipv6
+    # Checker
+    $$cap_appname-checker:
+        restart: always
+        volumes:
+            - $$cap_appname-checker-data:/usr/local/searx-checker/html/data
+        caproverExtra:
+            notExposeAsWebApp: true
+            dockerfileLines:
+                - FROM searx/searx-checker@$$cap_checker_version
+                - CMD -o html/data/status.json http://srv-captain--$$cap_appname:8080
+caproverOneClickApp:
+    variables:
+        - id: $$cap_searx_version
+          label: Searx Version Tag
+          defaultValue: '0.17.0-281-219af243'
+          description: Check out their Docker page for the valid tags https://hub.docker.com/r/searx/searx/tags/
+        - id: $$cap_filtron_version
+          label: Filtron Docker Version Hash
+          defaultValue: 'sha256:5b55f79b06fdd58122a6d764eca09d2ec687c5af5b78969124e69714d6bb889b'
+          description: Starts with 'sha256:'.  Find on https://hub.docker.com/r/dalf/filtron/tags/
+        - id: $$cap_morty_version
+          label: Morty Docker Version Hash
+          defaultValue: 'sha256:cb930c25254337e7a3abaf6daab326c663a4cdf1cbf4ae040ff7f670493df7fd'
+          description: Starts with 'sha256:'.  Find on https://hub.docker.com/r/dalf/morty/tags/
+        - id: $$cap_checker_version
+          label: Searx-checker Docker Version Hash
+          defaultValue: 'sha256:8ebda54ed8854e1300a4670e2bba86a5d2f4ceeb5c39290e4cc20df22bf974be'
+          description: Starts with 'sha256:'.  Find on https://hub.docker.com/r/searx/searx-checker/tags/
+        - label: Searx base url
+          description: Base url to your Searx
+          defaultValue: https://searx.app.gocloud.fun
+          id: $$cap_SEARX_BASEURL
+        - label: Morty URL
+          description: Base url to your Morty
+          defaultValue: https://searx-morty.app.gocloud.fun
```

Comment:
Not sure what "Morty" is. But this default value seems be just an example, right? If that's the case, move it to description please.

The idea behind default values is that if the user doesn't make any changes to them, they just work.

Reply:
See https://github.com/dalf/morty 

The main goal of morty is to provide a result proxy for searx

Features:
- HTML sanitization
- Rewrites HTML/CSS external references to locals
- JavaScript blocking
- No Cookies forwarded
- No Referrers
- No Caching/Etag
- Supports GET/POST forms and IFrames
- Optional HMAC URL verifier key to prevent service abuse

example

https://searx-morty.app.gocloud.fun/?mortyurl=https%3A%2F%2Fmedium.com%2Fswlh%2Fcaprover-the-definitive-guide-90076405aae4&mortyhash=dde3bfae15fc28782232c2cde20e9cc5697fa8b182279fdcad7ff8d7ff3d9437